test/atat-iam.test.ts

```diff
@@ -23,3 +23,233 @@ describe("ATAT network creation", () => {
     }
   });
 });
+
+describe("ATAT IAM Policy creation", () => {
+  let app: cdk.App;
+  let stack: cdk.Stack;
+  let template: Template;
+
+  beforeEach(() => {
+    app = new cdk.App();
+    stack = new AtatIam.AtatIamStack(app, "TestIamStack", {});
+    template = Template.fromStack(stack);
+  });
+
+  test("Fully assert generalReadAccess IAM managed policy with matchers", async () => {
```

Comment:
I really think this test is too specific. _any_ change we make to IAM permissions (adding or removing) is going to require a change to this test. That doesn't necessarily feel correct. We probably want to think more about what the API of the IAM stack is and test that, rather than just a raw CloudFormation template diff.

Reply:
This is what I was equivocating about as well. I was trying to understand the best practices a little deeper, whether there are use cases where we would actually want to do a raw CFN template diff, etc.

Reply:
The last thing I want is it to become an albatross.